### DISCUSSION TITLE
请问这是龙猫开发的PCL吗，看着咋那么不像呢（）

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1765] [提问] [answered]
打开文件看看吧……
[mcpclqdq.zip](https://github.com/Hex-Dragon/PCL2/files/11369923/mcpclqdq.zip)

我之前心血来潮想到PCL2前面有没有PCL1，就偶然下到这个文件的

**Comment by WForst-Breeze** [comment, marked as answer]
是的，这是 PCL1，不过现在只是一个能跑的状态，大部分功能无法正常使用（

**Reply by iPodtouch6557** [reply]
哦哦哦，谢谢，下载游戏时总是下到一半提示下载失败……
